CONTRIBUTING.md

```
@@ -9,7 +9,7 @@ First off, thank you for considering contributing to The Agency! It's people lik
 - [Agent Design Guidelines](#agent-design-guidelines)
 - [Pull Request Process](#pull-request-process)
 - [Style Guide](#style-guide)
-- [Community](#community)
+- [Community](#questions)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)